dev-lang/lazarus/lazarus-3.2_bin.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="Feature rich visual programming environment emulating Delphi"
+DESCRIPTION="Lazarus is a Rapid Application Development Tool for Free Pascal. It comes with the \
+LCL - Lazarus component library, which contains platform independent visual components like \
+buttons, windows, checkbox, treeview and many, many more. The LCL is platform independent, so you \
+can write an application once and then compile for various platforms without changing code.
+
+Free Pascal https://www.freepascal.org is a fast Object Pascal compiler, that runs on more than \
+20 platforms (Linux, Windows, BSD, OS/2, DOS, PowerPC,and many more)."
+HOMEPAGE="https://www.lazarus-ide.org/"
+COPYRIGHT="1993-2024 Lazarus and Free Pascal Team"
+LICENSE="GNU GPL v2
+	GNU LGPL v2"
+REVISION="1"
+srcGitRev="f6190f96e084f51d58124235201ce8f3fb5d7bb9"
+SOURCE_URI="https://codeberg.org/Begasus/Lazarus_Haiku/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="de2d420dd5b1824a44d43e97455a333c1460125087ec739d0ad2db806a4dae03"
+SOURCE_FILENAME="lazarus-v$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="lazarus_haiku/lazarus-${portVersion/\_bin/}"
+ADDITIONAL_FILES="lazarus.rdef.in"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	lazarus = $portVersion
+	cmd:lazarus_ide = $portVersion
+	cmd:lazbuild = $portVersion
+#	cmd:lazres = $portVersion
+#	cmd:lrstolfm = $portVersion
+	cmd:startlazarus = $portVersion
+#	cmd:updatepofiles = $portVersion
+	"
+REQUIRES="
+	haiku
+	fpc_source
+	cmd:fpc
+	lib:libiconv
+	lib:libQt5Pas
+	"
+
+INSTALL()
+{
+	mkdir -p $binDir $dataDir
+	cp -a data/* $dataDir
+	ln -sf $dataDir/lazarus/lazarus $binDir/lazarus-ide
+	ln -sf $dataDir/lazarus/lazbuild $binDir/lazbuild
+#	ln -sf $dataDir/lazarus/tools/lazres $binDir/lazres
+#	ln -sf $dataDir/lazarus/tools/lrstolfm $binDir/lrstolfm
+	ln -sf $dataDir/lazarus/startlazarus $binDir/startlazarus
+#	ln -sf $dataDir/lazarus/tools/updatepofiles $binDir/updatepofiles
```

Comment:
Are these binaries to be installed to `$dataDir`? That normally shouldn't contain any binaries, they go to `$binDir` or `$appsDir` or maybe `$libDir` (see libexec) if they are internal and not supposed to be run directly.

Is the reason that they must be kept together with their data files? Otherwise they should just be moved to `$binDir` instead of linking them, IMHO. Another alternative would be to use `$appsDir` instead of `$dataDir` in general.

Reply:
From their makefile:
```
	$(MKDIR) $(INSTALL_PREFIX)/share
	$(MKDIR) $(INSTALL_PREFIX)/share/lazarus
	$(MKDIR) $(INSTALL_PREFIX)/share/applications
	$(MKDIR) $(INSTALL_PREFIX)/share/pixmaps
	$(MKDIR) $(INSTALL_PREFIX)/share/mime/packages
	$(MKDIR) $(INSTALL_PREFIX)/share/icons/hicolor/48x48/mimetypes
	$(MKDIR) $(INSTALL_BINDIR)
	$(MKDIR) $(INSTALL_MAN_DIR)
	$(MKDIR) $(INSTALL_MAN_DIR)/man1
```
So they used "share" for those files, mostly we use "data" in those cases, so I used that.